feat(KaizenProvider): add KaizenProvider to @kaizen/components (#3239)
cultureamp/kaizen-design-system: packages/design-tokens/docs/pages/how-to-use-in-react.stories.mdx

# How to use design tokens in React components

> ## 1. Setup a KaizenProvider

Reviewer: ```suggestion
## 1. Set up a KaizenProvider
```

> Ensure you have followed the steps in [KaizenProvider](/story/components-kaizenprovider--page) to setup your application for using themes.

Reviewer: ```suggestion
Ensure you have followed the steps in [KaizenProvider](/story/components-kaizenprovider--page) to set up your application for using themes.
```

## 2. Start using design tokens with the `useTheme` hook

Once the theme context is available, the `useTheme` hook will allow you to subscribe components and give access to the current theme, and re-render whenever it changes.

Example:

```jsx
import { useTheme } from "@kaizen/design-tokens"

const Component = () => {
  const theme = useTheme();

  return (
    <div style={{
      backgroundColor: theme.color.purple["500"] // #844587
    }}>

>       Theme'd component

Reviewer: ```suggestion
      Themed component
```
I just can't get past it :') Any way to restructure so we use `theme` and not some variation?
